src/main/java/org/folio/service/orders/utils/ProductIdUtils.java

```diff
@@ -47,17 +47,17 @@ public static List<ProductId> removeISBNDuplicates(List<ProductId> productIds, S
   }
 
   public static  String extractProductId(String value) {
-    if(value == null || !value.contains(" ")){
+    if(value == null || !value.contains(" ")) {
       return value;
     }
-    return value.substring(0, value.indexOf(" "));
+    return value.split(" ")[0];
   }
 
   public static  String extractQualifier(String value) {
-    if(value == null || !value.contains(" ")){
+    if(value == null || !value.contains(" ")) {
       return null;
     }
-    return value.substring(value.indexOf(" ") + 1);
+    return value.split(" ")[1];
```

Comment:
Can 'qualifier' field contain " " ? In docs it is written as follows
![image](https://github.com/folio-org/mod-orders/assets/112848437/0c5a1aae-7100-40c8-bc22-2410165a6749)
If yes split() might not work as expected.